qa/tasks/cephfs/test_volumes.py

```diff
@@ -824,6 +826,13 @@ def test_clone_subvolume_info(self):
         # remove snapshot
         self._fs_cmd("subvolume", "snapshot", "rm", self.volname, subvolume, snapshot)
 
+        subvol_info = json.loads(self._get_subvolume_info(self.volname, subvolume))
+        if len(subvol_info) == 0:
```

Comment:
I think here and elsewhere in test_clone_subvolume_info and test_subvolume_info we should use,
```
self.assertEqual(actual result, expected result)
```
So this can be,
```
self.assertEqual(len(subvol_info), 0)
```

Reply:
Done